src/jit/lsra.cpp

```diff
@@ -1341,22 +1376,59 @@ compareBlocksForSequencing(BasicBlock* block1, BasicBlock* block2)
 //    as we encounter a block whose successors have all been sequenced, in pred-first
 //    order, or the very next block if we are traversing in random order (once implemented).
 //    This method uses a comparison method to determine the order in which to place
-//    the blocks in the list.  Currently, this is a simplistic method based on
-//    bbNum.  This could be a future candidate for tuning.
+//    the blocks in the list. 
```

Comment:
You should expand this comment to mention that we only query whether all predecessors have been sequenced **at the time it is added to the list** - this means that it will never get its priority/order bumped up once its remaining predecessors have been sequenced. Also, because different sorting criteria are being used to add blocks to the work list, we will wind up with a work list that is not really sorted in a consistent way. I think that's probably OK, but it should be noted.